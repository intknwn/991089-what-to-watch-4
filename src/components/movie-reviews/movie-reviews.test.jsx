import React from 'react';
import renderer from 'react-test-renderer';
import MovieReviews from './movie-reviews.jsx';

const reviews = [{
  id: 1,
  user: {
    id: 1,
    name: `Kate Doughlas`,
  },
  rating: 8.9,
  comment: `Awesome movie!`,
  date: `2019-05-08`,
},
{
  id: 2,
  user: {
    id: 2,
    name: `Bill Westmoore`,
  },
  rating: 8.0,
  comment: `Fantastic! Best movie ever!`,
  date: `2015-11-18`,
},
{
  id: 3,
  user: {
    id: 3,
    name: `Amanda Rockefeller`,
  },
  rating: 3.0,
  comment: `Bummer! The worst movie ever!`,
  date: `2015-11-18`,
}];

it(`MoviePage component should render page with movie details`, () => {
  const tree = renderer.create(
      <MovieReviews
        reviews={reviews}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
