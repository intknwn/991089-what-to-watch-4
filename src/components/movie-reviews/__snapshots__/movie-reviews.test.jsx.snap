// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePage component should render page with movie details 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Awesome movie!
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Kate Doughlas
          </cite>
          <time
            className="review__date"
            dateTime="2019-05-08"
          >
            May 8, 2019
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        8,9
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Fantastic! Best movie ever!
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Bill Westmoore
          </cite>
          <time
            className="review__date"
            dateTime="2015-11-18"
          >
            November 18, 2015
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        8
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Bummer! The worst movie ever!
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Amanda Rockefeller
          </cite>
          <time
            className="review__date"
            dateTime="2015-11-18"
          >
            November 18, 2015
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        3
      </div>
    </div>
  </div>
</div>
`;
